### PR TITLE
LPA code refactoring + endpoint integration

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

- Integrating case officer review outcome POST with API endpoint
- Refactoring to bring LPA code in line with new patterns and conventions introduced since work on this domain began (+ associated updates to templates

## Issue ticket number and link

N/A

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

### CODE REVIEW

File: apps/api/src/server/app/case-officer/case-officer-review.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ const reviewComplete = function (body) {
 	return Object.keys(body.reason).every((index) => !body.reason[index])? true : false;
 };
 
-// TODO: Refactor this to make it more readable. 
+// TODO: Refactor this to make it more readable.
 const invalidWithoutReasons = function (body) {
 	return ((
-		(body.reason.applicationPlansToReachDecisionMissingOrIncorrect  === true &&
-		body.reason.applicationPlansToReachDecisionMissingOrIncorrectDescription === undefined) ||
+		(body.reason.applicationPlansUsedToReachDecisionMissingOrIncorrect  === true &&
```
[the remark this answers is not in the document]
Comment (mayya-bond-kc) [reply apps/api/src/server/app/case-officer/case-officer-review.js:15]: we would need to change the database column names to match. consistency with what are you looking for?

[the remark this answers is not in the document]
Comment (james-hill-kc) [reply apps/api/src/server/app/case-officer/case-officer-review.js]: Ah no worries, I've changed it back. It was just for consistency with the labels shown in the design, but it's not important.

File: apps/api/src/server/app/case-officer/case-officer-review.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const reviewComplete = function (body) {
 	return Object.keys(body.reason).every((index) => !body.reason[index])? true : false;
 };
 
-// TODO: Refactor this to make it more readable. 
+// TODO: Refactor this to make it more readable.
```
Comment (james-hill-kc) [inline apps/api/src/server/app/case-officer/case-officer-review.js:12]: Not sure why this is showing as changed, maybe I removed a space or something

File: apps/web/src/server/app/lpa/lpa-session.service.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/**
+ * Reset the LPA state in the session.
+ *
+ * @param {object} session – The session containing an LPA state.
+ * @returns {void}
+ */
+export const destroy = (session) => {
+	delete session.lpa;
+};
+
```
Comment (james-hill-kc) [inline apps/web/src/server/app/lpa/lpa-session.service.js:10]: I didn't have chance to add the documenting comments yet, apologies - I wanted to get the functionality itself working before I finish on the project

**Review by radum — APPROVED** [review]

**Pull request merged**